stored_queries/generic_ontology/generic_ontology_get_ancestors.yaml

```diff
@@ -0,0 +1,43 @@
+# Get all ancestors (all parent's parents) for this term
```

Comment:
@zhlu9890 just to make this a little less long to read and write, can you rename the files from `generic_ontology` to just `ontology`?
For example: `generic_ontology/generic_ontology_get_ancestors.yaml` -> `ontology/ontology_get_ancestors.yaml`

Reply:
I agree with @jayrbolton.  I would rename these all and drop the generic.